<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>蓝牙列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_sousuo">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_zhuangtai">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>未连接</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_lanya">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>71</y>
     <width>339</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_wendu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>实时数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_lianjie">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>660</y>
     <width>93</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_duankai">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>660</y>
     <width>93</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="read_plot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(231, 206, 252)  ; /* 设置边框宽度为2像素，颜色为紫色*/
        border-radius: 20px; /* 设置四角为10像素的弧</string>
   </property>
   <property name="title">
    <string>实时动态曲线显示</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="realtime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#realtime
{
border: 4px solid rgb(231, 206, 252)  ; /* 设置边框宽度为2像素，颜色为紫色*/
border-radius: 20px; /* 设置四角为10像素的弧
}</string>
   </property>
   <property name="title">
    <string>实时数据</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="voltage">
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>电压:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="temp_num">
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>0.000</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="v">
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QGroupBox</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
